test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -72,6 +72,7 @@ internal static string TargetFramework
             if (!suppressBuildServer)
             {
                 buildArgumentList.Add($"/p:_RazorBuildServerPipeName={buildServerPipeName ?? BuildServer.PipeName}");
+                buildArgumentList.Add($"/p:_RazorSuppressCurrentUserOnlyPipeOptions=true");
```

Comment:
Can we add a comment here that says this is because our tests target `netcoreapp2.0`? Also can we file an issue that says we should upgrade tests to target `netcoreapp2.1`?

Reply:
Will do.